infrastructure/json/src/main/java/tech/pegasys/teku/infrastructure/json/types/CoreTypes.java

```diff
@@ -29,6 +30,15 @@
 
   public static final StringValueTypeDefinition<Byte> BYTE_TYPE = new ByteTypeDefinition();
 
+  public static final StringValueTypeDefinition<Bytes> BYTES_TYPE =
+      DeserializableTypeDefinition.string(Bytes.class)
+          .formatter(Bytes::toHexString)
+          .parser(Bytes::fromHexString)
+          .example("0xcf8e0d4e9587369b2301d0790347320302cc0943d5a1884560367e8208d920f2")
+          .description("Bytes hexadecimal")
+          .format("byte")
+          .build();
+
```

Comment:
I'm really not keen on an unbounded bytes type still. In the spec `BLS_WITHDRAWAL_PREFIX` is bytes1, so i think we use that to read it out, and maybe at some point we need to cleanup the code to not use bytes for it... would be interested in @ajsutton take on `BLS_WITHDRAWAL_PREFIX`

Reply:
Not sure I'd want to add a `Bytes1` class for this but yeah probably makes sense for the API to be able to specify that it's a single length byte.  No reason that can't be done with the existing `Bytes` class - just need to verify there is only one byte when parsing and serialising.

Reply:
maybe what we should do is just define this internally to spec and have a parser...
I'd really like to not have this unbounded type defined as a core type.